### DISCUSSION TITLE
Set kubeapi-server extra args

### DISCUSSION THREAD

**Discussion by AliyevH** [vitobotta/hetzner-k3s#176] [Q&A]
Hi. 
I could not find example how to set kube_api_server_args properly.
For example. I want to create k3s in hetzner and enable audit logging.
I have 2 files webhook and audit policy yaml files.

I have to add these lines to config file.

kube_api_server_args:
- audit-policy-file=audit-policy.yaml
- audit-log-path=webhook-config.yaml

Will script upload these files to master nodes? How it works?

**Comment by AliyevH** [comment]
@vitobotta

**Comment by sabuhish** [comment]
Hi @vitobotta,

I am also running k3s in hetzner, we need to configure to enable audit logging, the above issue blocks us. Do we have an update on this?
